core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1005,23 +1056,171 @@ private void processBind(@Transient ObjList<BindVariableSetter> bindVariableSett
             lo += Short.BYTES;
             bindVariables(lo, msgLimit, parameterCount, bindVariableSetters);
         }
-    }
+        prepareBindComplete();
+    }
+
+    private void processClose(long lo, long msgLimit, CharSequenceObjHashMap<NamedStatementWrapper> namedStatementMap) throws BadProtocolException {
+        final byte type = Unsafe.getUnsafe().getByte(lo);
+        if (type == 'S') {
+            lo = lo + 1;
+            long hi = getStringLength(lo, msgLimit);
+            checkNotTrue(hi == -1, "bad prepared statement name length");
+            CharSequence statementName = getStatementName(lo, hi);
+            if (statementName != null) {
+                final NamedStatementWrapper wrapper = namedStatementMap.get(statementName);
+                wrapper.clear();
+                bindVarTypesStack.push(wrapper.bindVariableTypes);
```

Comment:
wrapper.clear() sets bindVariableTypes to null. This code proceeded to fill stack with nulls. This seems rushed and untested.